src/main/java/warmup/Texas.java

```diff
@@ -0,0 +1,263 @@
+package main.java.warmup;
+// Here are warmup-1 practice tasks;
+
+public class Texas {
+    public static void main(String[] args) {
+        System.out.println("Texas");
+    }
+
+    //sleapIn enchanced
+    public boolean sleepIn(boolean weekday, boolean vacation) {
+        return !weekday || vacation;
+    }
+
+    //monkey trouble
+    public boolean monkeyTrouble(boolean aSmile, boolean bSmile) {
+        if (aSmile && !bSmile || !aSmile && bSmile) {
+            return false;
+        }
+        return true;
+    }
+
+    //sumDouble
+    public int sumDouble(int a, int b) {
+        if (a == b) {
+            return (a + b) * 2;
+        }
+        return (a + b);
+    }
+
+    //sumDouble solution #2
+    public int sumDouble2(int a, int b) {
+        return a == b ? (a + b) * 2 : a + b;
+    }
+
+    //diff21
+    public int diff21(int n) {
+        if (n < 21) {
+            return 21 - n;
+        }
+        return (n - 21) * 2;
+    }
+
+    //diff21 solution #2
+    public int diff212(int n) {
+        return n < 21 ? 21 - n : (n - 21) * 2;
+    }
+
+    //parrotTrouble enchanced
+    public boolean parrotTrouble(boolean talking, int hour) {
+        return talking && (hour < 7 || hour > 20);
+    }
+
+    //makes10 enchanced
+    public boolean makes10(int a, int b) {
+        return (a + b) == 10 || a == 10 || b == 10;
+    }
+
+    //nearHundred enchanced
+    public boolean nearHundred(int n) {
+        return Math.abs(n - 100) <= 10 || Math.abs(n - 200) <= 10;
+    }
+
+    //posNeg
+    public boolean posNeg(int a, int b, boolean negative) {
+        return a < 0 && b > 0 && !negative || a > 0 && b < 0 && !negative || a < 0 && b < 0 && negative;
+    }
+
+    //posNeg solution #2
+    public boolean posNeg2(int a, int b, boolean negative) {
+        if (a > 0 && b < 0 && !negative) {
```

Comment:
You return true in 3 branches.
It is fine, but can be a bit better.
Either merge three branches with 'return true' into one or simplify the last branch to something like that:

`        if (a > 0 && b < 0 && !negative) {
            return true;
        } else if (a < 0 && b > 0 && !negative) {
            return true;
        } else {
            return a < 0 && b < 0 && negative;
        }`